tools/pelican/content/ipynb/tutorial-installation.ipynb

```diff
@@ -0,0 +1,12866 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# OSCOVIDA Tutorial: installation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use OSCOVIDA in the cloud\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use OSCOVIDA in the cloud, you can go to [this link](https://mybinder.org/v2/gh/oscovida/binder/master?filepath=ipynb), \n",
+    "and can see all available Notebooks (most of which analyses one region, but you can also find the tutorial notebooks that start with the name `tutorial`). Or you can create your own notebook at that point.\n",
+    "\n",
+    "What technically happens is that (through the [binder project](http://mybinder.org)) a container (a bit like a small virtual machine) is created just for you at that point on a server somewhere on the planet. You can use this container from your browser, but any commands you execute are actually executed on this server somewhere else (i.e. \"in the cloud\"). You can use this container as long as you like, but there is a time-out: after 10 minutes of inactivity, your session will be closed.\n",
+    "\n",
+    "This way of using OSCOVIDA in the cloud is very convenient, because you don't need to install any software on your own computer/tablet etc.\n",
+    "\n",
+    "However, there are a number of limitations to be aware:\n",
+    "\n",
+    "- any interactive session you have, will be deleted after 10 minutes of inactivity\n",
+    "- at that point, your notebooks is also lost\n",
+    "- if you want to keep the notebook, you can go to `File -> Download as -> Notebook (.ipynb)` before your session expires\n",
+    "- if you have have saved such a notebook, you can upload it and then continue to work on it in your next session."
```

Comment:
Might be worth noting that some of the binder services may block downloads for security reasons, at least until we find a solution for that issue.